app/controllers/moderations_controller.rb

```diff
@@ -43,12 +43,12 @@ def index
     # filter based on type of thing moderated
     @what.each do |type, checked|
       next if checked
-      @moderations = @moderations.where("`moderations`.`#{type.to_s.singularize}_id` is null")
+      @moderations = @moderations.where("#{type.to_s.singularize}_id": nil)
```

Comment:
This is what squishes the brakeman warning

Reply:
Thanks for highlighting this, made it real clear what's happening.